internal/core/src/exec/expression/BinaryRangeExpr.cpp

```diff
@@ -740,10 +740,11 @@ PhyBinaryRangeFilterExpr::ExecRangeVisitorImplForJsonForIndex() {
                                       .clone();
         cached_index_chunk_id_ = 0;
     }
+    int total_data_chunk_pos= GetProcessedGlobalPos();
```

Comment:
We might as well record the position of the global pos, thus eliminating the need for recalculation each time.